lib/nylas/draft.rb

```diff
@@ -30,13 +30,23 @@ class Draft
 
     has_n_of_attribute :events, :event
     has_n_of_attribute :files, :file
+    has_n_of_attribute :file_ids, :string
     attribute :folder, :folder
     has_n_of_attribute :labels, :label
 
     attribute :tracking, :message_tracking
 
     transfer :api, to: %i[events files folder labels]
 
+    def create
```

Comment:
Do we also need to support this for update call?

I am also thinking of making `files` attribute as read_only so it won't get send to server. I think currently setting `files` with array of file object will send `files` key to server and API won't accept it and it might return error in future.

Reply:
Good call on both. 
- `files` is now read-only
- Added the extracting `file_id` to both save and update functions